src/main/scala/tilelink/Fuzzer.scala

```diff
@@ -179,45 +180,45 @@ class TLFuzzer(
     val a_type_sel  = noiseMaker(3, inc, 0)
 
     val legal = legal_dest && MuxLookup(a_type_sel, glegal, Seq(
-      UInt("b000") -> glegal,
-      UInt("b001") -> (pflegal && !Bool(noModify)),
-      UInt("b010") -> (pplegal && !Bool(noModify)),
-      UInt("b011") -> (alegal && !Bool(noModify)),
-      UInt("b100") -> (llegal && !Bool(noModify)),
-      UInt("b101") -> hlegal))
+      "b000".U -> glegal,
+      "b001".U -> (pflegal && !noModify.asBool),
+      "b010".U -> (pplegal && !noModify.asBool),
+      "b011".U -> (alegal && !noModify.asBool),
+      "b100".U -> (llegal && !noModify.asBool),
+      "b101".U -> hlegal))
 
     val bits = MuxLookup(a_type_sel, gbits, Seq(
-      UInt("b000") -> gbits,
-      UInt("b001") -> pfbits,
-      UInt("b010") -> ppbits,
-      UInt("b011") -> abits,
-      UInt("b100") -> lbits,
-      UInt("b101") -> hbits))
+      "b000".U -> gbits,
+      "b001".U -> pfbits,
+      "b010".U -> ppbits,
+      "b011".U -> abits,
+      "b100".U -> lbits,
+      "b101".U -> hbits))
 
     // Wire up Fuzzer flow control
-    val a_gen = if (nOperations>0) num_reqs =/= UInt(0) else Bool(true)
-    out.a.valid := !reset && a_gen && legal && (!a_first || idMap.io.alloc.valid)
+    val a_gen = if (nOperations>0) num_reqs =/= 0.U else true.B
+    out.a.valid := !reset.asBool && a_gen && legal && (!a_first || idMap.io.alloc.valid)
```

Comment:
I think this `.asBool` is actually correct

Reply:
Yes, I agree